src/ng/compile.js

```diff
@@ -2299,7 +2300,15 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     function assertNoDuplicate(what, previousDirective, directive, element) {
       if (previousDirective) {
         throw $compileMinErr('multidir', 'Multiple directives [{0}, {1}] asking for {2} on: {3}',
-            previousDirective.name, directive.name, what, startingTag(element));
+            getDirectiveLabel(previousDirective), getDirectiveLabel(directive), what, startingTag(element));
+      }
+      
+      function getDirectiveLabel(dir) {
+        if (dir.ngModule) {
```

Comment:
Forced to do this check because sometimes the directive function can be called without passing an `ngModule`.